packages/MainUI/src/components/FormView/selectors/QuantitySelector.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { memo, useState, useCallback, useEffect } from 'react';
+import { TextField } from '@mui/material';
+import { QuantityProps } from '../types';
+import { validateNumber } from '../../../utils/quantitySelectorUtil';
+
+const MAX_LENGTH = 9;
```

Comment:
Where is this value coming from exactly?

Reply:
This is not gonna be handled by the component, gonna be removed